gluten-core/src/main/scala/io/glutenproject/execution/WriteFilesExecTransformer.scala

```diff
@@ -51,7 +51,8 @@ case class WriteFilesExecTransformer(
     partitionColumns: Seq[Attribute],
     bucketSpec: Option[BucketSpec],
     options: Map[String, String],
-    staticPartitions: TablePartitionSpec)
+    staticPartitions: TablePartitionSpec,
+    var outputOrderOpt: Option[Seq[SortOrder]] = None)
```

Comment:
Thank you  for working on the improvement. But let's avoid adding workarounds to source code for passing UTs. Any issue if overriding Spark's UT checking method? It's `executeAndCheckOrdering` I see @ulysses-you had mentioned.